arrows/vtk/applets/color_mesh.cxx

```diff
@@ -398,13 +427,56 @@ class color_mesh::priv : public kwiver::arrows::vtk::mesh_coloration
       writer->Write();
       return true;
     }
+    else if (ext == ".las")
+    {
+      save_mesh_las(mesh, output_path);
+      return true;
+    }
     else
     {
       LOG_ERROR(main_logger, "Invalid file format: " << ext);
       return false;
     }
   }
 
+  void save_mesh_las(vtkSmartPointer<vtkPolyData> mesh,
```

Comment:
This function body should be in an ifdef checking if PDAL is enabled.  Thrown an exception or return false if it's not enabled.